charts/opentelemetry-collector/values.yaml

```diff
@@ -163,6 +163,8 @@ config:
           - zipkin
 
 image:
+  # Registry where to find the image
+  registry: docker.io
```

Comment:
I think we can continue to get by without any new fields here.  Since this chart only has one image if a user wants to specify a specific registry they can add it to `image.repository`.

For the subchart situation, I think we can update the template to prepend `global.imageRegistry` if it is present.

Reply:
I applied the suggestion or dropping `image.repository`.
But I left `global.imageRegistry` to follow the bitnami helm charts https://github.com/bitnami/charts/blob/main/bitnami/clickhouse/values.yaml#L15 for my usage of the `otel-collector` chart as a subchart